index.html

```diff
@@ -2,18 +2,19 @@
 
 <head>
   <title>Home page gallery</title>
+  <link rel="stylesheet" href="css/style.css">
 </head>
 <ul>
-  <li><a href="index.html">Home</a></li>
-  <li><a href="pages/pictures.html">Gallery</a></li>
-  <li><a href="pages/chloro.html">Chlorophytum data</a></li>
-  <li><a href="pages/sanseviera.html">Sansevieria data</a></li>
-  <li><a href="pages/Epipremnum.html">Epipremnum data</a></li>
+  <li class="page-link"><a href="index.html">Home</a></li>
+  <li class="page-link"><a href="pages/pictures.html">Gallery</a></li>
+  <li class="page-link"><a href="pages/chloro.html">Chlorophytum data</a></li>
+  <li class="page-link"><a href="pages/sanseviera.html">Sansevieria data</a></li>
+  <li class="page-link"><a href="pages/Epipremnum.html">Epipremnum data</a></li>
 </ul>
 
 <body>
 
-  <h2>The plants gallery :)</h2>
+  <h1 class="web-headline">The plants gallery :)</h1>
```

Comment:
why are you using a class here? seems like only this element is `web-headline`...

is there something else that can act "like" class but for only a single element?